js/src/redux/actions/snippetEditor.js

```diff
@@ -0,0 +1,50 @@
+export const SNIPPET_EDITOR_SWITCH_MODE = "SNIPPET_EDITOR_SWITCH_MODE";
+export const SNIPPET_EDITOR_UPDATE_DATA = "SNIPPET_EDITOR_UPDATE_DATA";
+export const SNIPPET_EDITOR_UPDATE_REPLACEMENT_VARIABLE = "SNIPPET_EDITOR_UPDATE_REPLACEMENT_VARIABLE";
+
+/**
+ * Switches mode of the snippet editor.
+ *
+ * @param {string} mode The mode the snippet editor should be in.
+ *
+ * @returns {Object} An action for redux.
+ */
+export function switchMode( mode ) {
+	return {
+		type: SNIPPET_EDITOR_SWITCH_MODE,
```

Comment:
In other files, we use things like 'SNIPPET_EDITOR' etc. as a prefix (see https://github.com/Yoast/yoast-components/blob/eb146b1b8570808192b2e9fc4994d390853bf10e/composites/Plugin/ContentAnalysis/actions/contentAnalysis.js) so the action names that are passed contain less redundant information

Reply:
I do not really like the prefix, but I'll change it for consistency.